Standalone/How to build nuget package.md

```diff
@@ -0,0 +1,11 @@
+﻿First build binaries of all supported platform. Enter
+```bat
+dotnet build -t:BatchBuild Steamworks.NET.Standard.sln
+```
+to your terminal.
+
+Then use nuget client with the `.nuspec` to pack an all-in-one package. Enter
+```bat
+nuget pack Steamworks.NET.nuspec -OutputDirectory bin\
+```
+to your terminal.
```

Comment:
Do you think we could have a build.bat and build.sh which runs these?

Reply:
Sure!